index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+  <nav class="container">
+    <img id="logo" src="img/image_geometry_1.png" alt="">
+    <div id="bars">
+      <a class="active" href="#"><i class="fa fa-bars" style="color:black;"></i></a>
+    </div>
+  </nav>
+  <header class="heading">
+    <div id="name">
+      Kashif Saleem
+    </div>
+  </header>
```

Comment:
- [ ]  For Making your HTML markup is well structured and follows a clear hierarchy and semantic correct I recommend you to use `header` tag for Toolbar section with in `nav` tag.

Reply:
Do you mean that I should replace the **nav** tag with a **header** tag?

Reply:
Hello @Kashif-Saleem-Ghuman `nav` tag should be inside the `header` tag.